lib/sqlalchemy/exc.py

```diff
@@ -188,6 +188,12 @@ def __init__(self, compiler, element_type, message=None):
             "Compiler %r can't render element of type %s%s"
             % (compiler, element_type, ": %s" % message if message else "")
         )
+        self.compiler = compiler
+        self.element_type = element_type
+        self.message = message
+
+    def __reduce__(self):
+        return self._class__, (self.compiler, self.element_type, self.message)
```

Comment:
**Federico Caselli** (CaselIT) wrote:

single _ here

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3107